lib/nolibc/README.md

```diff
@@ -0,0 +1,245 @@
+# nolibc, Unikraft's Lightweight Subset of the C Standard Library
+
+nolibc provides the basic core functionalities of libc in a lighter
+and more compact package, fulfilling the needs of most simple applications
+without occupying a large amount of memory, while also remaining POSIX-compliant.
```

Comment:
```suggestion
nolibc provides the basic core functionalities of libc in a lighter and more compact package, fulfilling the needs of most simple applications without occupying a large amount of memory, while also remaining POSIX-compliant.
```
Please use one sentence per line and one line per sentence, since it's easier to review.